web/src/models/__mocks__/Trace.mock.ts

```diff
@@ -13,6 +13,7 @@ const TraceMock: IMockFactory<TTrace, TRawTrace> = () => ({
         '1': SpanMock.raw(),
         '2': SpanMock.raw(),
       },
+      spans: [SpanMock.raw(), SpanMock.raw()],
```

Comment:
The raw trace doesn't have a spans array, it either comes from the flat or the tree section.